src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -362,7 +362,7 @@ public static int GetTrailingWhitespaceLength(this SyntaxNode node, FormattingCo
         // Tie-breaking.
         if (!getInnermostNodeForTie)
         {
-            var cuRoot = node.Ancestors().Last();
+            var cuRoot = node.Ancestors().LastOrDefault();
```

Comment:
If this is null, should we just return `node` here? This can only happen if the node is the root, right?

/cc @333fred